editor/doc/doc_data.cpp

```diff
@@ -982,8 +982,7 @@ Error DocData::save_classes(const String &p_default_path, const Map<String, Stri
 		FileAccessRef f = FileAccess::open(save_file, FileAccess::WRITE, &err);
 		if (err) {
 			ERR_EXPLAIN("Can't write doc file: " + save_file);
-
-			ERR_FAIL_V(err);
+			ERR_CONTINUE(err);
```

Comment:
That gets triggered when the target directory doesn't exist, and then all docs that come later in alphabetical order would be deleted... so this simple change prevents losing content. Will still throw an error though, I tried to add a `make_dir_recursive` to create the dir before retrying opening the `FileAccessRef`, but I got a crash I couldn't figure out and gave up :P